test/instructions.cc

```diff
@@ -19,6 +19,28 @@
 #define VALHALLA_SOURCE_DIR
 #endif
 
+// this is useful when you modify the Options proto and need to restore it
```

Comment:
as per usual, the canned protobufs are very annoying when changing the protobuf so i write a bit of code to help me modify them when i changed to a map from a repeated field. it could be useful in the future.